protostar/migrator/cheatcodes/migrator_declare_cheatcode.py

```diff
@@ -29,11 +29,33 @@ def __call__(
         ...
 
 
+class DeclareCheatcodeNetworkConfig(CheatcodeNetworkConfig):
```

Comment:
Note: We're going to pay for invoke in the future as well, so that's not going to be specific for this cheatcode only. Keep that in mind

Reply:
Currently, it is specific to this cheatcode hence I added it here. When it's shared, then we can easily refactor this code.
(Btw. I think NetworkConfig is a bad name. Maybe TxConfig would be better.)